charts/enterprise-logs/values.yaml

```diff
@@ -277,7 +277,9 @@ gateway:
   # -- Additional annotations for the `gateway` Pod
   annotations: {}
   # -- Additional labels and annotations for the `gateway` Service
+  # -- Service overriding service type 
```

Comment:
```suggestion
  # -- Service overriding service type
```
Looks like there's a trailing space here

Reply:
Fixed and pushed the change.